pylab/website/templates/website/event_details.html

```diff
@@ -43,4 +43,13 @@
 
 {{ event.description|markdown }}
 
+{% if request.user.is_authenticated %}
+  <div class="attendance">
+    <ul>
+  {% for attendee in attendance %}
+    <li>{{ attendee.attendee.username }} <span>{{ attendee.get_response_display }}</span></li>
```

Comment:
Use `username` template tag for rendering human friendly user name, like here: https://github.com/python-dirbtuves/website/blob/master/pylab/website/templates/base.html#L36